test/integration/targets/collections/posix.yml

```diff
@@ -22,6 +22,16 @@
       name: testns.testcoll.maskedmodule
     register: maskedmodule_out
 
+  # ensure the ansible ns can have real collections added to it
+  - name: call an external module in the ansible namespace
+    ansible.bullcoll.bullmodule:
+    register: bullmodule_out
+
+  # ensure the ansible ns cannot override ansible.builtin externally
```

Comment:
Why shouldn't it override the built-in? That's what happens with modules in roles today. It's helpful for bug fixes and features that won't be back ported.

Reply:
By design, collections don't allow overrides- the entire point of the `ansible.builtin` synthetic collection was to allow someone to say "I want the one in the box, not subject to whatever override shenanigans someone's doing". If you don't want that behavior, don't reference the `ansible.builtin` collection- the `ansible.legacy` synthetic collection is what you want (and what's implicitly there for search/resolution).